### ISSUE TITLE
[master, 5.x, 5.0] make logstash more chatty about errors on deb/rpm packages

### ISSUE THREAD

**Issue by jsvd** [elastic/logstash#5922]
currently, since the log4j changes in logging,  when launching logstash in the console, there is no logging stdout/stderr the presence of errors:

 ```
root@ip-10-77-33-21:~# /usr/share/logstash/bin/logstash -e "" --path.settings=/etc/logstash/
root@ip-10-77-33-21:~# 
root@ip-10-77-33-21:~# cat /var/log/logstash/logstash-plain.log 
figuration {:message=>"No config files found: /etc/logstash/conf.d/*\nCan you make sure this path is a logstash config file?"}
```

I understand that debian and rpm packages mean that most logging will go to `path.logs` but it's inconvenient that nothing is shown at all.

**Comment by talevy** [comment]
right, we don't ship console logging with the deb/rpm packaging. This was kind of done on purpose because it was decided that it shouldn't be used in the above manner.

**Comment by talevy** [comment]
fixed by #5999, right @jsvd?
